apps/ergw_core/src/gtp_context.erl

```diff
@@ -612,7 +612,7 @@ handle_event(info, {'DOWN', _MonitorRef, Type, Pid, _Info}, State,
     {next_state, shutdown, Data};
 
 handle_event({timeout, context_idle}, stop_session, State, Data) ->
-    delete_context(undefined, normal, State, Data);
+    delete_context(undefined, cp_inactivity_timeout, State, Data);
```

Comment:
Who wants/expects
cp_inactivity_timeout
It is not part of ergw.

Reply:
You are right it is not a part of erGW - this is a part of erGW-AAA and will used for termination cause mapping.